roles/setup-site/tasks/main.yml

```diff
@@ -48,9 +49,9 @@
     site_exists: FALSE
   when: search_for_site|length == 0
 
-# - name: debug
-#   debug:
-#     var: site_exists
+- name: debug
```

Comment:
I'd like to change our debug statements to only print when a variable is set to true.

Reply:
I didn't mean to leave this uncommented. Bad developer; no cookie.